### DISCUSSION TITLE
[2291850]Underground Blossom 本体

### DISCUSSION THREAD

**Discussion by BeaterLee** [pjy612/SteamManifestCache#76] [Share]
appid:2291850
code:46184E92DC307D8D
[Underground Blossom.zip](https://github.com/pjy612/SteamManifestCache/files/13695526/Underground.Blossom.zip)
求一个内测名额

**Comment by pjy612** [comment]
想想你缺点啥

**Reply by BeaterLee** [reply]
是appkey要写出来吗 我以为放压缩包就行 
keys:
2291851;7301f06ff6d9c682a664803eaaaa115ffb893380e1f2f7bb8a3e8d23490b4768

**Reply by ZEERDEER** [reply]
Email

**Reply by BeaterLee** [reply]
sorry，email：745972340@qq.com

**Reply by pjy612** [reply]
查收。key 和 app 放外面 最好 ，毕竟直观。
